### ISSUE TITLE
Screen does not regain full height after dismissing software keyboard (Samsung)

### ISSUE THREAD

**Issue by jay-jlm** [akveo/kittenTricks#280]
## 🐛 Bug Report

After focusing the text input in the messaging example screens, upon hiding the software keyboard (down arrow button), the screen does not stretch back to full size, but instead stays at the same height that it had when the keyboard was shown, taking up only half of the total screen real state.
The issue remains even if you navigate to other screens of the app.

## To Reproduce

Tested on Android Samsung Galaxy A10.
1) Access any of the chat message example screens
2) Tap the input field to bring up the native software keyboard.
3) Tap the down arrow button at the bottom or anywhere on the screen to dismiss the software keyboard.

## Expected behavior

After hiding the software keyboard the screen should expand back to original full height.

## UI Kitten and Eva version

Version 2.4.0 downloaded from the Play Store.

**Comment by joshbedo** [comment]
Having same issue on Google Pixel 3a.

**Comment by karthikcodes6** [comment]
Having the same issue on Honor 6x

**Comment by jrock17** [comment]
Same on Google Pixel 4
![bug-1](https://user-images.githubusercontent.com/1270524/93491900-12701a00-f8bf-11ea-82f4-ec8a987e8390.png)

![bug-2](https://user-images.githubusercontent.com/1270524/93491909-14d27400-f8bf-11ea-85aa-96cbdafebd4d.png)

**Comment by ClementVanPeuter** [comment]
Yeah same issue with Kitten Tricks for theses screens and same behavior for inputs when the page is scrollable.
What are the news about that ? It's just a kitten tricks Issue or a Kitten Ui too ?
